templates/tips/read_more.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+<div class="container">
+    <div class="grid">
+        <article class="post vt-post">
+            <div class="row">
+                <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4">
+                    <div class="post-type post-img">
+                        <a href="#"><img src="https://bootdey.com/img/Content/avatar/avatar1.png" class="img-responsive"
+                                alt="image post"></a>
+                    </div>
+                    <div class="author-info author-info-2">
+                        <ul class="list-inline">
+                            <li>
+                                <div class="info">
+                                    <p>Posted On:</p>
+                                    <strong>{{ tip.date}}</strong>
+                                </div>
+                            </li>
+                            <li>
+                                <div class="info">
+                                    <p>Category:</p>
+                                    <strong>{{ tip.category }}</strong>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-xs-12 col-sm-7 col-md-7 col-lg-8">
+                    <div class="caption">
+                        <h3 class="md-heading"><a href="#">{{ tip.author }}</a></h3>
+                        <p> {{ tip.text }}</p>
+                        <a class="edit" type="button" href="edit_tip/{{ tip.id }}" role="button">Edit</a><br />
+                    </div>
+                </div>
+            </div>
+        </article>
+        <div class="clearfix"></div>
+    </div>
+</div>
+</body>
+{% endblock %}
```

Comment:
missing line

Reply:
In html files the formating is without a new line I think